src/LanguageServer/Test/RdtTests.cs

```diff
@@ -63,5 +70,91 @@ public void TestInitialize()
             rdt.GetDocument(uri1).Should().BeNull();
             ds.Diagnostics.TryGetValue(uri1, out _).Should().BeFalse();
         }
+
+        [TestMethod, Priority(0)]
+        public async Task LockCount() {
+            var uri1 = TestData.GetDefaultModuleUri();
+            await CreateServicesAsync(PythonVersions.LatestAvailable3X, uri1.AbsolutePath);
+            var rdt = Services.GetService<IRunningDocumentTable>();
+
+            rdt.OpenDocument(uri1, "from LockCount1 import *");
+            var doc = rdt.GetDocument(uri1);
+            await doc.GetAstAsync(CancellationToken.None);
```

Comment:
You shouldn't need this line anymore.

Reply:
But I do. You may want to look into it. If I remove it, test fails b/c 
```
var docLc2 = rdt.First(d => d.Name.Contains("LockCount2"));
```
`Sequence contains no elements`